src/mapper/sensor_summary_mapper.py

```diff
@@ -13,3 +16,10 @@ def to_sensor(sensor_id):
         "tags": sensor.sensor_tags,
         "installTime": sensor.sensor_install_time
     }
+
+def to_sensor_detail(sensor):
+    return {
+        "id": str(sensor.id),
+        "sensor_status": sensor.sensor_status,
+        "tags": sensor.sensor_tags
+    }
```

Comment:
Not sure what is this mapping for? it looks like this method is used in sensor `controller`, but I don't see any objects in the swagger that has `sensor_status` field.

Reply:
Fixed, this is what is return when a new sensor is posted.